src/sys/kernel.rs

```diff
@@ -45,8 +45,13 @@ impl Version {
     ///
     /// ```
     pub fn from_str(s: &str) -> Result<Self, &'static str> {
-        let mut s = s.split('-');
-        let kernel = s.next().unwrap();
+        let pos = s.find(|c: char| c != '.' && ((c < '0') || (c > '9')));
```

Comment:
Could `! c.is_ascii_digit()` be used here?  It might be slightly more readable.

Reply:
Thanks. That's right.